lms/services/email_preferences.py

```diff
@@ -29,8 +32,40 @@ class EmailPrefs:
     sat: bool = True
     sun: bool = True
 
-    def days(self) -> dict:
-        return {key: value for key, value in asdict(self).items() if key in self.DAYS}
+    @staticmethod
+    def user_preferences_key_for_email_digest_date(datetime_: datetime) -> str:
+        return f"instructor_email_digests.days.{EmailPreferences.DAYS[datetime_.weekday()]}"
+
+    def unsubscribe_instructor_digest(self) -> Self:
+        self.mon = self.tue = self.wed = self.thu = self.fri = self.sat = self.sun = (
```

Comment:
This changes every digest setting to false.

It leaves any other setting intact (eg, when we have a mention one).